src/relcast.erl

```diff
@@ -343,70 +344,89 @@ deliver(Seq, Message, FromActorID, State = #state{in_key_count = KeyCount,
                                                  <<FromActorID:16/integer, Message/binary>>),
                     {ok, NewState#state{in_key_count = KeyCount + 1,
                                         transaction_dirty = true,
+                                        %% new_messages = true,
                                         defers = maps:put(FromActorID, [Key|N], Defers)}};
                 _ ->
                     %% sorry buddy, no room on the couch
                     full
             end
     end.
 
-%% default to false for ergonomics
 take(ID, State) ->
```

Comment:
all this junk is back compat for eqc, tests, and old libp2p, we can tear it out later.